gtest_parallel_mocks.py

```diff
@@ -48,14 +48,20 @@ def assertRecorded(self, test_id, expected, retries):
 
 
 class TestTimesMock(object):
-  def __init__(self, test_lib):
+  def __init__(self, test_lib, test_data=None):
```

Comment:
Should this default to `{}` or will that not generate a new dictionary per usage?

Reply:
It will re-use the same dictionary over multiple calls.

Reply:
Ew, nice foot shooter, Python.